README.md

```diff
@@ -127,6 +127,14 @@ ReactiveUI is developed under an OSI-approved open source license, making it fre
 
 If you want to submit pull requests please first open a [GitHub issue](https://github.com/reactiveui/ReactiveUI/issues/new/choose) to discuss. We are first time PR contributors friendly.
 
+The current source uses **Visual Studio 2022 Preview**. Please install the following workloads to enable building the source:
```

Comment:
You can run net 6 with the 2019 preview. It has to be preview though.

Reply:
Changed to vs2019 preview